client/contivModelClient.go

```diff
@@ -403,8 +403,17 @@ type PolicyLinks struct {
 	Tenant Link `json:"Tenant,omitempty"`
 }
 
+type PolicyOper struct {
+	Endpoints        []EndpointOper `json:"endpoints,omitempty"`
+	NumEndpoints     int            `json:"numEndpoints,omitempty"`     // number of endpoints
+	PolicyViolations int            `json:"policyViolations,omitempty"` // number of policyViolations
```

Comment:
List of endpoint groups using a policy is more useful information than the endpoints itself i think. From endpoint groups, you can get the endpoints anyway.. Lets return the list of endpoint groups using the policy here..
Also, please remove PolicyViolations from the PR since it needs more work..